src/styles/block-card/card.scss

```diff
@@ -0,0 +1,111 @@
+.card {
+  max-width: 200px;
+  border-radius: 5px;
+  border: 1px solid $background-color;
+
+  &:hover {
+    transform: scale(1.2);
+    transition: ease 300ms;
+  }
+
+  &__img {
+    display: flex;
+    justify-content: center;
+    padding: 32px 19px 0;
+    margin-bottom: 40px;
+
+    &-link {
+      height: 134px;
+    }
+  }
+
+  &__tittle {
+    margin-bottom: 4px;
+    padding-inline: 16px;
+    color: $main-accent;
+    font-size: 12px;
+    font-weight: 500;
+    font-style: normal;
+    line-height: 18px;
+
+    &:hover {
+      color: #34568b;
+      transition: ease 300ms;
+    }
+  }
+
+  &__code {
+    margin-bottom: 16px;
+    padding-inline: 16px;
+    color: $secondary-accent;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__reviews {
+    display: flex;
+    align-items: flex-end;
+    justify-content: space-between;
+    margin-bottom: 24px;
+    padding-inline: 16px;
+
+    &-text {
+      color: $main-accent;
+      font-size: 10px;
+      font-weight: 400;
+      font-style: normal;
+      line-height: 14px;
+      text-align: right;
+    }
+  }
+
+  &__price {
+    display: flex;
+    text-align: center;
+    justify-content: space-between;
+    margin-bottom: 16px;
+    padding-inline: 16px;
+
+    &-text {
+      color: $secondary-accent;
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+    }
+
+    &-value {
+      color: $main-accent;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &__button {
+    display: flex;
+    margin: 16px;
+    border-radius: 5px;
+    background-color: $button-accent;
+    align-items: center;
+
+    &-link {
+      padding: 12px 70px;
+      line-height: 16px;
+      font-size: 14px;
+      font-weight: 700;
+      text-transform: uppercase;
+      text-decoration: none;
+      color: $background-color;
+      text-align: center;
+
+      &:hover {
+        border: 1px solid $button-accent;
+        background-color: $background-color;
+        color: $button-accent;
+        border-radius: 5px;
+        transition: ease 300ms;
```

Comment:
same here with transition.

Reply:
Also set properties for transition explictly or use all key word and check [transition](https://developer.mozilla.org/en-US/docs/Web/CSS/transition) shorthand order